templates/quickstart-bitbucket-dc-with-vpc.template.yaml

```diff
@@ -3,33 +3,18 @@ AWSTemplateFormatVersion: 2010-09-09
 Description: "Atlassian Bitbucket Data Center in new VPC License: Apache 2.0"
 Metadata:
   AWS::CloudFormation::Interface:
-
     ParameterGroups:
       - Label:
           default: Bitbucket setup
         Parameters:
           - BitbucketVersion
-      - Label:
```

Comment:
Why shuffle all the parameters around?

Reply:
To make the two templates (with and without VPC) look as similar as possible in terms of structure and ordering